.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
change this to `.eslintrc.js` that uses `module.exports` like the prettier config.